x-pack/test/cases_api_integration/security_and_spaces/tests/common/comments/delete_comment.ts

```diff
@@ -387,35 +386,6 @@ export default ({ getService }: FtrProviderContext): void => {
         });
       });
 
-      it('should delete multiple comments from the appropriate owner', async () => {
```

Comment:
I moved all tests about `deleteAllComments` to `x-pack/test/cases_api_integration/security_and_spaces/tests/common/comments/delete_comments.ts`